### PR TITLE
Feature/df 1211  user changed event

### PR DESCRIPTION
**Pull request by emina-str**
other PR : https://github.com/alv-ch/jobroom2/pull/894

### CODE REVIEW

File: application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java [hunk 1]
```diff
@@ -64,15 +64,24 @@ public void handleUserRegisteredEvent(UserRegisteredEvent userRegisteredEvent) {
 		List<NPABusinessCase> npaBusinessCases = this.npaBusinessCaseRepository.findByPersonNumber(personNumber);
 		npaBusinessCases.forEach(npaBusinessCase -> {
 			Optional<ProofOfWorkEfforts> existingPoWE = this.proofOfWorkEffortsRepository.findByControlPeriod(personNumber, npaBusinessCase.getControlPeriod());
-			if (existingPoWE.isPresent()) {
-				existingPoWE.get().update(this.toNpaBusinessCaseTemplate(npaBusinessCase), this.dateCalculator);
-			} else {
+			if (!existingPoWE.isPresent()) {
```
Comment (pavax) [inline application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: this line is not related to this PR, or is it?

File: application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java [hunk 2]
```diff
@@ -64,15 +64,24 @@ public void handleUserRegisteredEvent(UserRegisteredEvent userRegisteredEvent) {
...
 				this.create(toCreateProofOfWorkEffortsDto(npaBusinessCase, userData));
 			}
 			this.npaBusinessCaseRepository.delete(npaBusinessCase);
 		});
 	}
 
+	@IsSysAdmin
+	public void handleUserChangedEvent(UserChangedEvent userChangedEvent) {
+		Page<ProofOfWorkEfforts> proofOfWorkEffortsList = this.proofOfWorkEffortsRepository.findByUserOwnerId(
```
Comment (pavax) [inline application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: should we use a stream instead of a page ?

File: application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java (hunk 1 above)
Comment (emina-str) [reply application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: oh, it was left from my previous testing and checking ..  i'll revert it.

File: application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java (hunk 2 above)
Comment (emina-str) [reply application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: added method in Repository that returns Stream of PoWE
